rotkehlchen/api/v1/schemas.py

```diff
@@ -319,7 +318,7 @@ class BlockchainTransactionDeletionSchema(Schema):
         exclude_types=(
             SupportedBlockchain.ETHEREUM_BEACONCHAIN,
             SupportedBlockchain.AVALANCHE,
-            *typing.get_args(SUPPORTED_BITCOIN_CHAINS),
+            SupportedBlockchain.BITCOIN_CASH,
```

Comment:
Why exclude bitcoin cash? Shouldn't what we do work for all bitcoin like chains (namely normal and cash?)

Reply:
We should also support BCH as it shoudl be trivial I guess, no?